plugin/src/index.ts

```diff
@@ -72,22 +72,17 @@ const plugin: NetlifyPlugin = {
 
     checkNextSiteHasBuilt({ publish, failBuild })
 
-    const {
-      appDir,
-      basePath,
-      i18n,
-      images,
-      target,
-      ignore,
-      trailingSlash,
-      outdir,
-      experimental: {
-        images: { remotePatterns },
+    let experimentalRemotePatterns = []
+    const { appDir, basePath, i18n, images, target, ignore, trailingSlash, outdir, experimental } = await getNextConfig(
+      {
+        publish,
+        failBuild,
       },
-    } = await getNextConfig({
-      publish,
-      failBuild,
-    })
+    )
+
+    if (experimental.images) {
+      experimentalRemotePatterns = experimental.images.remotePatterns
```

Comment:
[potential boulder] Is it possible that `remotePatterns` might not be defined even if `experimental.images` exists? 

If there's a dependency on `experimentalRemotePatterns` needing to be an array, then it might be worth doing the following as an added measure of safety

```suggestion
      experimentalRemotePatterns = experimental.images.remotePatterns ?? []
```